staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -1026,7 +1026,7 @@ func truncateBody(body string) string {
 // allocating a new string for the body output unless necessary. Uses a simple heuristic to determine
 // whether the body is printable.
 func glogBody(prefix string, body []byte) {
-	if klog.V(8) {
```

Comment:
same here... logging all request bodies is far more verbose than other V(3) usage